mods/d2k/rules/structures.yaml

```diff
@@ -593,7 +593,7 @@ large_gun_turret:
 		QuantizedFacings: 32
 	Armament:
 		Weapon: TowerMissile
-		LocalOffset: 256,384,768, 256,-384,768
+		LocalOffset: 256,0,768
```

Comment:
This one - haven't checked the missile tank yet - doesn't have burst on the weapon, which is why the swapping launchers back and forth is changed to a "fire between the two" look. Another polish could be halving missile damage and giving it Burst: 2 to maintain the image of using both launchers.